sdk/messaging/azservicebus/internal/stress/tests/finite_peeks.go

```diff
@@ -51,9 +53,13 @@ func FinitePeeks(remainingArgs []string) {
 	sc.PanicOnError("failed to abandon message",
 		receiver.AbandonMessage(sc.Context, tmp[0], nil))
 
-	for i := 0; i < 10000; i++ {
-		log.Printf("Sleeping for 1 second before iteration %d", i)
-		time.Sleep(time.Second)
+	maxPeeks := 10000
```

Comment:
constants?

Reply:
I don't know why I didn't do that already...